src/d3a/d3a_core/sim_results/cumulative_grid_trades.py

```diff
@@ -15,16 +15,40 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-from d3a.d3a_core.sim_results.area_statistics import accumulate_grid_trades_all_devices, \
-    generate_cumulative_grid_trades_for_all_areas
-from d3a_interface.constants_limits import ConstSettings
+from d3a.constants import FLOATING_POINT_TOLERANCE
+from d3a.d3a_core.util import area_name_from_area_or_iaa_name, add_or_create_key, \
+    make_iaa_name_from_dict, subtract_or_create_key, round_floats_for_ui
 
 
-def export_cumulative_grid_trades(area, accumulated_trades_redis, past_market_types):
-    accumulated_trades = \
-        accumulate_grid_trades_all_devices(area, accumulated_trades_redis, past_market_types)
-    return accumulated_trades, generate_cumulative_grid_trades_for_all_areas(accumulated_trades,
-                                                                             area, {})
+def _is_cell_tower_type(area):
+    return area['type'] == "CellTowerLoadHoursStrategy"
+
+
+def _is_load_node_type(area):
```

Comment:
I see that these functions have been renamed from `_is_load_node` to `is_load_node_type`, but the old functions `is_load_node` are still there. Therefore, either the old function definitions need to be deleted if they are not needed anymore in the area_statistics module, or these functions need to be moved to a separate module (maybe the __init__) in order to be reused by all the modules that might need them

Reply:
Yes, moved all `*_type` method to __init__.py module, though keeping the old implementation `is_load_node ` as still being used in some places for instance for kpis and will completely deprecate once all the subtasks of this parent story has been completed